README.rst

```diff
@@ -11,48 +11,93 @@ Usage
 
 .. code-block:: bash
 
-    wget --output-document=upstream-control "https://raw.githubusercontent.com/GoogleCloudPlatform/guest-agent/main/packaging/debian/control"
-    wget --output-document=ubuntu-control "https://git.launchpad.net/ubuntu/+source/google-guest-agent/plain/debian/control?h=applied/ubuntu/noble-devel"
-    python3 google_guest_agent_packaging_diff_tool.py --upstream-control-file ./upstream-control --ubuntu-control-file ./ubuntu-control
+    python3 google_guest_agent_packaging_diff_tool.py --agent-package google-osconfig-agent
 
-TODO
-----
+The current output is as follows:
+
+.. code-block:: bash
 
-This project is very much in progress and is currently in POC stage. The following are the things that need to be done:
+        $ python3 google_guest_agent_packaging_diff_tool.py --agent-package google-compute-engine-oslogin
```

Comment:
Just want to be sure that the argument to `--agent-package` here is meant to be different from the preceding example above.

Reply:
It was, but I've changed it again in the next upload as I've edited the formatting a bit more :laughing: